src/clr-angular/forms/combobox/utils/combobox-dom-adapter.service.ts

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+ * Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
```

Comment:
nit: Services should go into the providers directory.

Reply:
Moved them